.github/workflows/check_if_pr_is_automergeable.yml

```
@@ -45,20 +68,28 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       issues: write
-    needs: check_test_results
-    if: needs.check_test_results.outputs.ALL_TESTS_PASS == 'True'
+    needs: check_pr_details
+    if: needs.check_pr_details.outputs.ALL_TESTS_PASS == 'True'
     steps:
       - name: Check out repository code
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Get PR number from workflow context
-        run: |
-          echo "PR_NUMBER=$( python brainscore_vision/submission/actions_helpers.py get_pr_num )" >> $GITHUB_ENV
       - name: Add automerge-approved label to PR
         env:
           GH_TOKEN: ${{ secrets.WORKFLOW_TOKEN }}
           GH_REPO: ${{ github.repository }}
-          NUMBER: ${{ env.PR_NUMBER }}
+          NUMBER: ${{ needs.check_pr_details.outputs.PR_NUMBER }}
           LABELS: automerge-approved
         run: gh issue edit "$NUMBER" --add-label "$LABELS"
+  
+  notify_failure:
+    name: If any test fails, notify the user through the brain-score email account (only needed for web submissions)
+    uses: ./.github/workflows/user_notification_system.yml
+    needs: check_pr_details
+    if: needs.check_pr_details.outputs.ALL_TESTS_PASS == 'Failure' && needs.check_pr_details.outputs.AUTOMERGE_WEB == 'true'
```

Can the value of `ALL_TESTS_PASS` be kept consistent (ie either `True` or `False` instead of `Failure`)?

```suggestion
    if: needs.check_pr_details.outputs.ALL_TESTS_PASS == 'Failure' && needs.check_pr_details.outputs.AUTOMERGE_WEB == 'true'
```